statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -626,6 +802,12 @@ def __init__(self, model):
         # Setup the cache
         self._cache = resettable_cache()
 
+        # Setup covariance matrix notes dictionary
+        if not hasattr(self, 'cov_kwds'):
+            self.cov_kwds = {}
+        self.cov_type = 'opg'
```

Comment:
> We do, but the problem is that I can't use the inherited LikelihoodModelResults machinery for cov_type because if cov_type is set it wants to run the model through get_robustcov_results.

But I should have it as a keyword argument to the MLEResults class that I strip out prior to calling `super`, I'll make that change.